### PR TITLE
[doc] Udpate document about filter subplugins

### PR DESCRIPTION
**Pull request by anyj0527**
- Update some outdated docs about tensor_filter subplugins

### CODE REVIEW

File: Documentation/component-description.md [hunk 1]
```diff
@@ -62,14 +62,17 @@ In this page, we focus on the status of each elements. For requirements and desi
...
   - openVINO/Intel (stable)
   - ARMNN (stable)
-  - SNPE/Qualcomm (stable)
+  - SNPE/Qualcomm (stable for both v1 and v2)
+  - QNN/Qualcomm (stable)
   - Vivante/Verisilicon (stable)
   - TensorRT/NVidia (stable)
   - SNAP (stable)
   - Deepview-RT/NXP (stable. **maintained by the manufacturer**)
+  - OnnxRuntime (stable)
```
Comment (jaeyun-jung) [inline Documentation/component-description.md:71]: plz check name (DeepViewRT, ONNX Runtime)

File: Documentation/features-per-distro.md [hunk 2]
```diff
@@ -27,6 +26,7 @@ This document reflects the status of 2022-07-14 (2.1.1 devel)
         - ARMNN
         - TVM
         - SNPE
+        - onnxruntime
```
Comment (jaeyun-jung) [inline Documentation/features-per-distro.md:29]: ditto

File: Documentation/features-per-distro.md [hunk 3]
```diff
@@ -54,16 +54,17 @@ This list shows a release corresponding to "full".
     - ROS extensions are **not** released.
 - Extra Elements and Subplugins
     - Enabled
-        - tensorflow2-lite (2.3, 2.7.0, 2.8.1) w/ delegation support (GPU, XNNPACK, NNAPI, ...) since 2.7.0
+        - tensorflow2-lite (2.8.1, 2.16.1) w/ delegation support (GPU, XNNPACK, NNAPI, QNN, ...)
         - nnfw runtime (a.k.a. ONERT)
         - (depending on options) snap / snpe (you cannot install both simultaneously)
+        - QNN
```
Comment (jaeyun-jung) [inline Documentation/features-per-distro.md:60]: 'qnn' for sub-plugin name?

File: gst/nnstreamer/tensor_filter/README.md [hunk 4]
```diff
@@ -30,7 +30,7 @@ More launch line examples here: [nnstreamer example](https://github.com/nnstream
 - Framerate policies
 - Recurrent network support
 - Supported framework
-  - TensorFlow, TensorFlow-lite, NNFW(ONE), Caffe2, Python 3, PyTorch, OpenVINO, EdgeTPU, ArmNN, TensorRT, SNPE, SNAP
+  - TensorFlow, TensorFlow-lite, NNFW(ONE), Caffe2, Python 3, PyTorch, OpenVINO, EdgeTPU, ArmNN, TensorRT, SNPE, SNAP, QNN, ExecuTorch, MXNet, NCNN, onnxruntime, DeepView RT, dali.
```
Comment (jaeyun-jung) [inline gst/nnstreamer/tensor_filter/README.md:33]: ditto (DeepViewRT, ONNX Runtime, DALI)

File: Documentation/component-description.md (hunk 1 above)
Comment (jaeyun-jung) [reply Documentation/component-description.md:71]: DALI/NVidia ?

File: gst/nnstreamer/tensor_filter/README.md (hunk 4 above)
Comment (anyj0527) [reply gst/nnstreamer/tensor_filter/README.md]: I updated those terms as you suggest, thanks!

**Review by songgot — APPROVED** [review]
LGTM

**Review by jaeyun-jung — APPROVED** [review]

**Review by gichan-jang — APPROVED** [review]
LGTM

**Pull request merged**